server/middlewares/validePasswordReset.js

```diff
@@ -0,0 +1,81 @@
+import jwt from 'jsonwebtoken';
```

Comment:
Delete this file because It is not being used anywhere else. As I remember, We have imported its functionality in the authHelper instead. But double-check to confirm that.

Reply:
Resolved